diff --git a/ragel/goipgoto.cpp b/ragel/goipgoto.cpp
index 55d1891..359c986 100644
--- a/ragel/goipgoto.cpp
+++ b/ragel/goipgoto.cpp
@@ -324,7 +324,7 @@ std::ostream &GoIpGotoCodeGen::FINISH_CASES( int level )
 
 			/* Write each action in the eof action list. */
 			for ( GenActionTable::Iter item = act->key; item.lte(); item++ )
-				ACTION( out, item->value, STATE_ERR_STATE, true, false );
+				ACTION( out, item->value, STATE_ERR_STATE, true, true );
 		}
 	}
 
